release-notes/unreleased/2870-POI-filters-fix-toggle-button.yml

```
@@ -0,0 +1,6 @@
+issue_key: 2870
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - android
+  - ios
+en: A Fix for the toggle button at the poi filter.
```

```suggestion
en: A fix for the toggle button in the map filter
de: Der Toggle-Button im Kartenfilter wurde verbessert
```
We call them pois internally but most users probably don't know that that means point of interest.